scripts/utils/create_package_template.py

```diff
@@ -327,8 +438,8 @@ def main(argv=None):
     parser.add_argument("--category", type=str, choices=CATEGORIES, help="Category for tool")
     parser.add_argument("--description", type=str, default="", help="Description for tool")
     parser.add_argument("--dependency", type=str, default="", help="Metapackage dependency")
-    parser.add_argument("--zip_url", type=str, default="", help="URL to ZIP file")
-    parser.add_argument("--zip_hash", type=str, default="", help="SHA256 hash of ZIP file")
+    parser.add_argument("--target_url", type=str, default="", help="URL to target file (zip or executable)")
+    parser.add_argument("--target_hash", type=str, default="", help="SHA256 hash of target file (zip or executable)")
     parser.add_argument("--shim_path", type=str, default="", help="Metapackage shim path")
```

Comment:
Do we need to add `--target_url_64` and `--target_hash_64` here?

Reply:
Based on https://docs.chocolatey.org/en-us/create/functions/get-chocolateywebfile#url-string it seems we're *supposed* to use the correct URL based on the bitness of the tool we're downloading. But in practice, VMs will likely be 64-bit and I believe the default `-url` parameter should be fine. So we could technically remove the `x64` counterparts from the code to download in the template. Thoughts?

Reply:
I'll leave them there for now, but won't add the `_64` arguments here.